.gitignore

```diff
@@ -5,3 +5,4 @@ vendor
 composer.phar
 composer.lock
 phpunit.xml
+.idea
```

Comment:
This file should never be included in .gitignore. Please, use a global gitignore in your local :)